traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -1011,7 +1011,7 @@ <h3>Current Value</h3>
                     <div class="form-group" ng-class="{'has-error': hasError(routingConfig.maxDnsAnswers), 'has-feedback': hasError(routingConfig.maxDnsAnswers)}">
                         <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="maxDnsAnswers">Max DNS Answers<div class="helptooltip">
                             <div class="helptext">
-                                Max number of IP addresses in DNS answer (0 means all)
+                                If the Traffic Router profile parameter "edge.http.limit" is set, setting this to a non-zero value will override that parameter for this delivery service, limiting the number of Traffic Router IP addresses (A records) that are included in responses to DNS requests for this delivery service.
```

Comment:
want to just point this to https://traffic-control-cdn.readthedocs.io/en/latest/overview/delivery_services.html#max-dns-answers as well and simply have 2 definitions in the docs?

Reply:
ok.. so segregate based on DNS and HTTP/Steering in the RST docs?

Reply:
yes

Reply:
otherwise, someone using the api and not TP will not be aware of the different behavior.

Reply:
okie.. I shall update.